addons/project/static/src/scss/project_kanban.scss

```diff
@@ -0,0 +1,5 @@
+.o_kanban_examples_dialog {
+    .o_notebook_headers {
+        width: 22rem;
+    }
+}
```

Comment:
I am wondering if the issue is not in Framework side. 🤔 

I see the `overflow-x: auto;` has been added to avoid showing the tabs in the form view when they overtakes the form view. But here in the examples, the notebook tabs are displayed vertically (instead of horizontally in the from view), so maybe we could avoid adding `overflow-x: auto;` when the notebook headers are displayed vertically?

If yes, we could do this instead: 

```diff
diff --git a/addons/web/static/src/core/notebook/notebook.scss b/addons/web/static/src/core/notebook/notebook.scss
index ffc98357b7e5..a89d2d850c2a 100644
--- a/addons/web/static/src/core/notebook/notebook.scss
+++ b/addons/web/static/src/core/notebook/notebook.scss
@@ -8,7 +8,6 @@

     .o_notebook_headers {
         margin: 0 var(--notebook-margin-x, 0);
-        overflow-x: auto;

         @include media-breakpoint-down(md) {
             &::-webkit-scrollbar {
diff --git a/addons/web/static/src/core/notebook/notebook.xml b/addons/web/static/src/core/notebook/notebook.xml
index 755e13e0a1d6..010b06bc2a60 100644
--- a/addons/web/static/src/core/notebook/notebook.xml
+++ b/addons/web/static/src/core/notebook/notebook.xml
@@ -3,7 +3,7 @@

     <t t-name="web.Notebook" owl="1">
         <div t-attf-class="o_notebook d-flex w-100 {{ props.orientation === 'horizontal' ? 'horizontal flex-column' : 'vertical flex-row' }} {{ props.className }}" t-if="state.currentPage">
-            <div class="o_notebook_headers" t-att-class="{ 'm-0': props.orientation === 'vertical' }">
+            <div t-attf-class="o_notebook_headers {{ props.orientation === 'vertical' ? 'm-0' : 'overflow-auto' }}">
                 <ul t-attf-class="nav nav-tabs {{ props.orientation === 'horizontal' ? 'flex-row flex-nowrap' : 'flex-column p-0' }}">
                     <li t-foreach="navItems" t-as="navItem" t-key="navItem[0]" class="nav-item flex-nowrap cursor-pointer" t-if="navItem[1].isVisible">
                         <a class="nav-link" t-attf-class="{{ navItem[0] === state.currentPage ? 'active' : '' }} {{ props.orientation === 'vertical' ? 'p-3 rounded-0' : '' }} {{ navItem[1].className }}" t-att-name="navItem[1].name" t-on-click.prevent="() => this.activatePage(navItem[0])" href="#" role="tab" tabindex="0" t-esc="navItem[1].title"/>
```

What do you think @odoo/rd-framework-js ? 🙂

Reply:
Like @xavierbol said, I don't think either that it's a project's issue but more an inconsistency in the notebook's styles.

Although, pay attention that the utility class `overflow-auto` targets both direction, and is not an equivalent to `overflow-x: auto`.

Instead I would suggest to simply do the following :
```patch
diff --git a/addons/web/static/src/core/notebook/notebook.scss b/addons/web/static/src/core/notebook/notebook.scss
index ffc98357b7e5..03159af90ed4 100644
--- a/addons/web/static/src/core/notebook/notebook.scss
+++ b/addons/web/static/src/core/notebook/notebook.scss
@@ -48,6 +48,10 @@
     }
 
     &.vertical {
+        .o_notebook_headers {
+            overflow-x: visible;
+        }
+
         .nav {
             width: max-content;
             border-bottom-color: transparent; // removing the width has weird side effect due to the negative margin
```

It will counterbalance the `overflow-x: auto;` rule used in the general (aka. horizontal) use case, and force the overflowing element to take the space it needs, adjusting the headers column's width accordingly (as it has a `width: max-content` rule).

Reply:
Done! :+1: 

We actually initially thought it not an issue from notebook but intended behavior from notebook and it is not something we are expecting in `o_kanban_examples_dialog` so we decide to give specific max width for that particular scenario

Reply:
Oh indeed sorry @pparidans 😅 

Thanks for your review. 🙂